.circleci/config.yml

```diff
@@ -1,29 +1,52 @@
 version: 2
 
+common-steps:
+  - &bootstrap
+    run:
+      name: Install sudo, git and make
+      command: apt-get update && apt-get install -y sudo git make
```

Comment:
Does sphinx uses git for something? CircleCI doesn't need it for the special `checkout` step IIRC.

If you get rid of the make target (see my comment there), you don't need sudo either.

Reply:
Ah, I assumed `checkout` needed it - it does not! Removed.